priv/repo/migrations/20201210043230_create_notes.exs

```diff
@@ -0,0 +1,19 @@
+defmodule ChatApi.Repo.Migrations.CreateNotes do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes, primary_key: false) do
+      add :id, :binary_id, primary_key: true
+      add :body, :text
+      add :author_id, references(:users, on_delete: :nothing, type: :id)
+      add :account_id, references(:accounts, on_delete: :nothing, type: :binary_id)
```

Comment:
let's do `on_delete: :delete_all` here -- if an account is deleted, we want all the associated notes to be deleted as well